subsys/partition_manager/Kconfig

```diff
@@ -4,6 +4,25 @@
 # SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
 #
 
+menu "Partition Manager"
+
+menu "Zephyr subsystem configurations"
+
+if FILE_SYSTEM_LITTLEFS
+partition=LITTLEFS
+partition-size=0x6000
+source "${ZEPHYR_BASE}/../nrf/subsys/partition_manager/Kconfig.template.partition_size"
```

Comment:
We are already inside the `nrf/subsys/partition_manager/` here, so why refer back to `${ZEPHYR_BASE}` ?

Reply:
There were some issues with how paths were resolved in these situation earlier, but I can see if this has now been fixed so that we can use a simpler path.